### DISCUSSION TITLE
Editor play testing gives no starting room

### DISCUSSION THREAD

**Discussion by Morilli** [ppy/osu#21264] [Ideas] [answered]
If a beatmap starts at 0:000, play testing the map feels frustrating because it's not possible to hit the first couple of notes due to the editor jumping in at exactly 0:000, giving no lead-in time like playing normally would.

https://user-images.githubusercontent.com/35152647/202346674-b7916570-50d1-4b69-aaed-09ec753dbae8.mp4

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/pull/21265
